packages/core-flows/src/handlers/sales-channel/create-sales-channels.ts

```diff
@@ -0,0 +1,35 @@
+import { createStep, StepResponse } from "@medusajs/workflows-sdk"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  CreateSalesChannelDTO,
+  ISalesChannelModuleService,
+  SalesChannelDTO,
+} from "@medusajs/types"
+
+export const createSalesChannelsStep = createStep<
+  CreateSalesChannelDTO[],
+  SalesChannelDTO[],
+  string[]
```

Comment:
todo: this should be removed, any particular reason to do that and not rely on the inferece?